common/src/main/java/io/netty5/util/internal/PlatformDependent.java

```diff
@@ -826,6 +826,17 @@ public static boolean useDirectBufferNoCleaner() {
         return USE_DIRECT_BUFFER_NO_CLEANER;
     }
 
+    /**
+     * Get the ID of the given thread. This is {@code Thread.threadId()} on Java 19 and newer,
+     * or {@code Thread.getd()} on older Java versions.
```

Comment:
it seems a typo in the java doc `{@code Thread.getd()}` -> `{@code Thread.getId()}`